plugins/module_utils/network/ios/config/snmp_server/snmp_server.py

```diff
@@ -163,6 +212,13 @@ def generate_commands(self):
         wantd = self._snmp_list_to_dict(self.want)
         haved = self._snmp_list_to_dict(self.have)
 
+        # Take in count the traps config mpls_vpn which is DEPRECATED and replaced by mpls.vpn
+        if "traps" in wantd and "mpls_vpn" in wantd["traps"]:
```

Comment:
Could you please make this into a separate function called handle_deprecates, also could you pop the mpls_vpn out of the dict at the end